integration-tests/infinispan-common/src/test/java/org/apache/camel/quarkus/component/infinispan/common/InfinispanCommonServerTestResource.java

```diff
@@ -46,8 +47,8 @@ public Map<String, String> start() {
                 .withExposedPorts(HOTROD_PORT)
                 .withEnv("USER", USER)
                 .withEnv("PASS", PASS)
-                .withClasspathResourceMapping("infinispan.xml", "/user-config/infinispan.xml", BindMode.READ_ONLY)
-                .withCommand("-c", "/user-config/infinispan.xml")
+                .withClasspathResourceMapping("camel-infinispan-cache.xml", "/user-config/camel-infinispan-cache.xml", BindMode.READ_ONLY)
+                .withCommand("-c infinispan.xml -c /user-config/camel-infinispan-cache.xml")
```

Comment:
we can provide several chunks (in json, xml or yaml), avoiding the need of a full xml file that will potentially break the container when updating